src/elementor/infrastructure/request-post.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Yoast\WP\SEO\Elementor\Infrastructure;
+
+use WP_Post;
+
+/**
+ * Retrieve the WP_Post from the request.
+ */
+class Request_Post {
```

Comment:
It might be nice to disable the WordPress.Security.NonceVerification.Recommended and InputNotSanitized PHPCS checks for the entire class because now they just take up a lot of space in the class making it less readable.